source/math/FloatMatrix.ooc

```diff
@@ -219,44 +224,27 @@ FloatMatrix : cover {
 	// </summary>
 	// <param name="y">The right hand column y vector of the equation system.</param>
 	// <returns>Return the least square solution to the system.</returns>
-	solve: func@ (y: This) -> This {
+	solve: func (y: This) -> This {
 		result: This
 		if (this width > this height)
 			raise("Invalid dimensions in FloatMatrix solve")
 		// TODO: This can probably be cleaned up...
```

Comment:
Why fix something and leave the TODO? It here will only cause confusion, indicating there is something to be fixed here when there no longer is.